metabrainz/static/css/main.less

```diff
@@ -211,6 +211,26 @@ h1.page-title, h2.page-title {
   }
 }
 
+//team page 
+#team {
+  strong { color: @green; }
+  .people { border-bottom: 2px solid @green; }
```

Comment:
It is an element for separating content and this is what the goal is here, I assume. I'm not saying `hr` needs to be used there. I'm saying that styling of separators should be consistent throughout the whole website.